rasa/nlu/featurizers/dense_featurizer/convert_featurizer.py

```diff
@@ -17,15 +20,33 @@
     DENSE_FEATURIZABLE_ATTRIBUTES,
     FEATURIZER_CLASS_ALIAS,
     TOKENS_NAMES,
+    NUMBER_OF_SUB_TOKENS,
 )
-from rasa.shared.nlu.constants import TEXT, FEATURE_TYPE_SENTENCE, FEATURE_TYPE_SEQUENCE
+from rasa.shared.nlu.constants import (
+    TEXT,
+    FEATURE_TYPE_SENTENCE,
+    FEATURE_TYPE_SEQUENCE,
+    ACTION_TEXT,
+)
+from rasa.exceptions import RasaException
+import rasa.nlu.utils
 import numpy as np
 import tensorflow as tf
 
 import rasa.utils.train_utils as train_utils
 
 logger = logging.getLogger(__name__)
 
+# URL to the old remote location of the model which
+# users might use. The model is no longer hosted here.
+ORIGINAL_TF_HUB_MODULE_URL = (
+    "https://github.com/PolyAI-LDN/polyai-models/releases/download/v1.0/model.tar.gz"
+)
+
+# Warning: This URL is only intended for running pytests on ConveRT
+# related components. This URL should not be allowed to be used by the user.
+RESTRICTED_ACCESS_URL = "https://storage.googleapis.com/continuous-integration-model-storage/convert_tf2.tar.gz"
```

Comment:
Can't users simply download the model from here and use the model than locally? Should we somehow protect this URL a bit more?

Reply:
cc @dakshvar22 
I guess this should be tackled in a separate PR if we want to change it.

Reply:
We discussed a few other approaches briefly but decided to go this route because others didn't really seem feasible. See this [slack thread](https://rasa-hq.slack.com/archives/C36SS4N8M/p1603736235407200).